docs/superpowers/plans/2026-06-06-pr-23-artifact-release-tooling.md

```
@@ -0,0 +1,1996 @@
+  --output release/artifacts/manifests/manifest.json \
+  --base-url https://artifacts.example.test
+```
+
+Validate a local archive against a release record:
+
+```shell
+cargo run -p pv-release -- validate-archive \
+  --archive path/to/artifact.tar.gz \
+  --record release/artifacts/records/path/to/record.json
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)